x-pack/plugins/security_solution/server/lib/siem_migrations/rules/siem_rule_migrations_service.ts

```diff
@@ -43,29 +65,16 @@ export class SiemRuleMigrationsService {
     assert(currentUser, 'Current user must be authenticated');
     assert(this.esClusterClient, 'ES client not available, please call setup first');
 
-    const esClient = this.esClusterClient.asScoped(request).asCurrentUser;
-    const dataClient = this.rulesDataStream.createClient({ spaceId, currentUser, esClient });
+    // TODO: change to `esClient = this.esClusterClient.asScoped(request).asCurrentUser;` when the API is made public
```

Comment:
The tickets for siem migrations are still in a private repo, so instead of referencing a private repo in the code I did it the other way around. I created the ticket and referenced the code [here](https://github.com/elastic/security-team/issues/11176).